Exibe os DOIs de acordo com as diretrizes do Crossref

joffilyfe opened the pull request:
#### O que esse PR faz?
Este pull request realiza alterações pontuais nos arquivos que de alguma maneira exibem os `DOIs` dos artigos. As páginas de texto completo, "como citar" e "nlinks" foram modificadas.

#### Onde a revisão poderia começar?
- `htdocs/xsl/sci_common.xsl` L: `1098`

#### Como este poderia ser testado manualmente?
Para testar este PR manualmente, deve-se:
- Crie uma instância do SciELO metodologia [[2]](###comandos);
- Atualize as bases de acordo com seu teste (ex: as bases mst, xml de scielo br e medline);
- Navegue entre artigos, verifique os links de `DOI`;
- Verifique os link em `"como citar"`;
- Verifique os links na página `arttext_plus`;

#### Algum cenário de contexto que queira dar?

Não foram aplicados estilos em cascata para alterar a exibição, as cores, tamanhos ou fontes dos links. Sendo assim os links não terão uma aparência especial.

### Screenshots
![Screen Shot 2020-02-11 at 15 27 29](https://user-images.githubusercontent.com/4604104/74268167-2549bd00-4ce6-11ea-97d8-2cad157cc4af.png)
![Screen Shot 2020-02-11 at 15 28 21](https://user-images.githubusercontent.com/4604104/74268173-2844ad80-4ce6-11ea-8009-d450f41f4731.png)


#### Quais são tickets relevantes?
N/A

### Referências
[[1] - Crossref DOI Guidelines_pt](https://docs.google.com/document/d/1K3Lj031Q6LqYxo33Byh0EzywskO-ywkKg526ENqYa54/edit#)

### Comandos
[2] - Criando uma nova instância de scielo metodologia

```bash
export SCIELO_FOLDER="$HOME/metodologia/dados-master"

docker run --name scielo-master -v "$SCIELO_FOLDER/scielo":/var/www/scielo \
    -v "$SCIELO_FOLDER/logs":/var/www/apache \
    --env INSTANCE_NAME=scielo_scl_master \
    --env USER_SUPERVISOR=root \
    --env PASS_SUPERVISOR=toor \
    --env SITE_NAME=localhost:8080 \
    --env USER_FTP=teste \
    --env PASSWD_FTP=teste \
    --env USER_PASS=scielo \
    --env GIT_BRANCH_NAME=master \
    -p 8080:8080 -p 2223:2222 -d scieloorg/metodologia:latest
```

gustavofonseca commented:
Ainda não consigo aplicar o patch no meu repositório local.

```bash
gustavofonseca@srm-matriz-old:Web (rev708) git apply --check /tmp/708.patch
error: patch failed: cgi-bin/ScieloXML/sci_isoref.xis:274
error: cgi-bin/ScieloXML/sci_isoref.xis: patch does not apply
error: patch failed: cgi-bin/ScieloXML/sci_nlinks.xis:194
error: cgi-bin/ScieloXML/sci_nlinks.xis: patch does not apply
error: patch failed: cgi-bin/ScieloXML/sci_reflinks.xis:164
error: cgi-bin/ScieloXML/sci_reflinks.xis: patch does not apply
error: patch failed: htdocs/xsl/plus/data.xsl:347
error: htdocs/xsl/plus/data.xsl: patch does not apply
error: patch failed: htdocs/xsl/plus/layout.xsl:231
error: htdocs/xsl/plus/layout.xsl: patch does not apply
error: patch failed: htdocs/xsl/sci_common.xsl:1095
error: htdocs/xsl/sci_common.xsl: patch does not apply
```

joffilyfe commented:
@gustavofonseca não sei o porque deste erro no seu repositório local. Tentei reproduzir da seguinte forma:
1) Clonei o repositório em um novo path;
2) Verifiquei se o patch é aplicável `joffily@lb-rancher ~/Web> git apply --check  708.patch` (não deu erro);
3) Apliquei `git apply 708.patch`;

O resultado foi:

```bash
joffily@lb-rancher ~/Web> git status                                                                                                                                                                                                                                   master!?
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   cgi-bin/ScieloXML/sci_isoref.xis
	modified:   cgi-bin/ScieloXML/sci_nlinks.xis
	modified:   cgi-bin/ScieloXML/sci_reflinks.xis
	modified:   htdocs/xsl/plus/data.xsl
	modified:   htdocs/xsl/plus/layout.xsl
	modified:   htdocs/xsl/sci_common.xsl

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	708.patch

no changes added to commit (use "git add" and/or "git commit -a")
```

gustavofonseca commented:
@joffilyfe, acredito que havia alguma sujeira na minha cópia local. Após eu limpá-la a aplicação do patch passou a ser possível. Malz pelo alarme falso rs.
